server/custom_vms.go

```diff
@@ -262,7 +304,7 @@ func (lc *localNetwork) waitForCustomVMsReady(
 					return errAborted
 				case <-ctx.Done():
 					return ctx.Err()
-				case <-time.After(10 * time.Second):
+				case <-time.After(1 * time.Second):
```

Comment:
Can we make this timeout a constant? Or at least put a comment explaining what this means?

Reply:
Also, this can just be `time.Second`, not `1 * time.Second`